services/galley/src/Galley/Data/Conversation/Types.hs

```diff
@@ -37,6 +37,7 @@ data Conversation = Conversation
     convName :: Maybe Text,
     convAccess :: [Access],
     convAccessRole :: AccessRole,
+    convAccessRoles :: Set AccessRoleV2,
```

Comment:
Same here: `convAccessRole` shouldn't be part of the type, the resp. database column should be translated away in the `Cql` instance.